CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 <!--
 	Placeholder for the next version (at the beginning of the line):
-	## __WORK IN PROGRESS__
+	## **WORK IN PROGRESS**
```

Comment:
this will crash the release pipleine

Reply:
Hm. I used it everywhere im my adapters. 50 times... Does the AlCalzone script used for release?